lang/cs/Org.Apache.REEF.IO.Tests/TestAzureBlockBlobFileSystemE2E.cs

```diff
@@ -295,6 +290,20 @@ public void TestDeleteDirectorySecondLevelE2E()
             Assert.True(CheckContainerExists(_container));
         }
 
+        private List<CloudBlockBlob> SetupBlobs(string directory)
```

Comment:
>SetupBlobs [](start = 37, length = 10)

nit: SetupBlobs doesn't really let me know what this function is doing. I suggest renaming something more explanitory and also exposing the blob count as well. maybe something like CreateTempBlobs(string directory, int blobCount=3)

Reply:
fixed

---
In reply to: [194555964](https://github.com/apache/reef/pull/1473#discussion_r194555964) [](ancestors = 194555964)